Product/Production/Services/CORE_X12DocumentSubmissionCore/pom.xml

```diff
@@ -51,6 +51,27 @@
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
         </dependency>
+        <!-- Testing -->
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.jmock</groupId>
```

Comment:
That's not a good reason to include them.  They're not being used and we've been using mockito instead of jmock as our preferred mocking library since release 4.0. Jmock is only there for older unit tests.